README.md

```diff
@@ -1,19 +1,48 @@
 # Ansible Playbook Bundle (APB)
 
-Ansible Playbook Bundle (APB) is a lightweight application definition (meta-containers). APB
-has the following features:
+An **Ansible Playbook Bundle (APB)** is a lightweight application definition (meta-container). They are used to define and deploy complex groups of applications, deployment configs, deployments, and services to an [OpenShift Origin](https://github.com/OpenShift/origin) cluster running the [Ansible Service Broker](https://github.com/fusor/ansible-service-broker).  APBs offer more power and  simple configuration by leveraging the power of Ansible. APBs have the following features:
```

Comment:
This is probably a larger discussion, but should we target kubernetes too? We don't need to have the discussion here, but I think mentioning it as a supported cluster alongside origin is important.